wandb/sdk/internal/internal_api.py

```diff
@@ -3590,28 +3640,40 @@ def fetch_org_entity_from_entity(self, entity: str) -> Tuple[str, str]:
                 "entityName": entity,
             },
         )
-        try:
-            is_team = response["entity"].get("isTeam", False)
-            org = response["entity"]["organization"]
-            org_name = org["name"] or ""
-            org_entity_name = org["orgEntity"]["name"] or ""
-        except (LookupError, TypeError) as e:
-            if is_team:
-                # This path should pretty much never be reached as all team entities have an organization.
+        if response is None:
+            raise ValueError(
+                f"Unable to find an entity with name: {entity!r}, response failed: {response}"
+            )
```

Comment:
question/suggestion: mentioned offline -- do we need this block?  If not (i.e. if a bad response status auto-raises an error), maybe we can remove this